lib/MusicBrainz/Server/Form/Role/ValidateUsername.pm

```diff
@@ -0,0 +1,22 @@
+package MusicBrainz::Server::Form::Role::ValidateUsername;
```

Comment:
I think a role could make sense for username behavior in general, so also adding `has_field 'username'` here at least, assuming the `required` attribute works in both places. But, just moving the function to lib/MusicBrainz/Server/Validation.pm is sane too.